### PR TITLE
 testing/py3-weasyprint: upgrade to 43; remove unsupported python2

### PR DESCRIPTION
**Pull request by stwa**
As stated on the news page, python2 is no longer supported.

https://weasyprint.org/news/#post-1
> Python 3.4+ is now needed, Python 2.x is not supported anymore

### CODE REVIEW

File: testing/py3-weasyprint/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Contributor: William Pitcock <nenolod@dereferenced.org>
+# Contributor: Stefan Wagner <stw@bit-strickerei.de>
+# Maintainer: William Pitcock <nenolod@dereferenced.org>
```
Comment (andypost) [inline testing/py3-weasyprint/APKBUILD]: needs @kaniini approaval & notes about why py2 unsupported (link)

**Comment by andypost** [comment]
please remove unrelated commits

**Comment by stwa** [comment]
Depends on https://github.com/alpinelinux/aports/pull/5679, https://github.com/alpinelinux/aports/pull/5680 and https://github.com/alpinelinux/aports/pull/5671.

**Comment by Ikke** [comment]
This seems to do partially the same as #5671.

Can you also rebase this PR on the latest master?

**Pull request closed without merging**